lib/circles/topics/create.js

```diff
@@ -0,0 +1,30 @@
+const stageEnum = require("./stageEnum.json");
+
+module.exports = function(req, res) {
+    const topicId = `${req.circle.circleId}-${req.body.title}`;
```

Comment:
I am not a big fan of using the title here, as it will probably contain several words and hence be bad in a URL ... (lots of `%20` up in here).

Could we auto-generate a unique id which is something like:

```js
const topicId = req.circle.circleId + "-" + req.body.title.toLowerCase().replace(/\s+/g, "-").replace(/[^a-z0-9\-]/g, "");
```

(The title is made lower case, has non-alphanumeric characters stripped and whitespace replaced with `-`).

The topic part of the ID should probably be saved to the topic, too. We would have a topicId (the last part) and a canonicalTopicId (the full part, prefaced with cirlce id).